torch/csrc/jit/script/compiler.cpp

```diff
@@ -715,6 +762,23 @@ struct to_ir {
           node->t_(Symbol::attr("alpha"), at::CPU(at::kFloat).scalarTensor(1.0));
         return node->outputs();
       }
+      case '*': {
```

Comment:
I think it would be good if we had some more control over where the `*` can appear, because it's very limited in Python, and we want to pretend we're a subset, not a superset